### ISSUE TITLE
Scroll Reverser works occasionally for trackpad 

### ISSUE THREAD

**Issue by bchewy** [pilotmoon/Scroll-Reverser#73]
Hi all,

I'm running on macOS Catalina 10.15.1 with the supplemental update 1; Scroll reverser use to work fine without the preferences changing. Occasionally when I'm using my mouse - it works just fine (with reversed scrolling) - **but when I use my trackpad to say move around and scroll - it gets reversed even though the reverse trackpad is left unchecked.**

Would appreciate any help!
Thank you!

**Comment by invariant** [comment]
Do you have Scroll Reverser enabled in the "Input Monitoring" category of Privacy settings?

**Comment by agergec** [comment]
Hello,

I'm using scroll reverser since Mojave. It was working great even in the first days of Catalina. But some unknown chage or system update broke it.

I noticed that toggling "Reverse Mouse" option behaves as the "Scroll direction" preference in System Preferences. If the Reverse Mouse is selected, both Mouse and Trackpad are reversed. If Reverse Mouse is unselected, both devices are natural.  Beisdes Reverse Trackpad option is not working.

Any assistance for troubleshooting will be greatly appreciated.

PS: I enabled Scroll Reverser, in Input Monitoring and Accesibility settings.

kind regards
Gökalp
